index.html

```diff
@@ -49,7 +49,7 @@ <h2>About-Me</h2>
     <section id="MyLife" class="#MyLife"></section>
```

Comment:
class names and id names should be in different names for style and review reasons :D

Reply:
and no # in classnames.

Reply:
Okay, that should have changed by now, thanks for pointing that out.
With the main, I really need to double-check how I do it.